src/panels/lovelace/components/hui-action-editor.ts

```diff
@@ -54,72 +54,131 @@ export class HuiActionEditor extends LitElement {
     if (!this.hass || !this.actions) {
       return html``;
     }
+
     return html`
-      <paper-dropdown-menu
-        .label="${this.label}"
-        .configValue="${"action"}"
-        @value-changed="${this._valueChanged}"
-      >
-        <paper-listbox
-          slot="dropdown-content"
-          .selected="${this.actions.indexOf(this._action)}"
+      <div class="dropdown">
+        <paper-dropdown-menu
+          .label=${this.label}
+          .configValue=${"action"}
+          @iron-select=${this._actionPicked}
         >
-          ${this.actions.map((action) => {
-            return html` <paper-item>${action}</paper-item> `;
-          })}
-        </paper-listbox>
-      </paper-dropdown-menu>
-      ${this._action === "navigate"
+          <paper-listbox
+            slot="dropdown-content"
+            attr-for-selected="value"
+            .selected=${this.config?.action ?? "default"}
+          >
+            <paper-item .value=${"default"}
+              >${this.hass!.localize(
+                "ui.panel.lovelace.editor.action-editor.actions.default_action"
+              )}</paper-item
+            >
+            ${this.actions.map((action) => {
+              return html`
+                <paper-item .value=${action}
+                  >${this.hass!.localize(
+                    `ui.panel.lovelace.editor.action-editor.actions.${action}`
+                  )}</paper-item
+                >
+              `;
+            })}
+          </paper-listbox>
+        </paper-dropdown-menu>
+        <ha-help-tooltip
+          .label=${this.hass!.localize(
+            "ui.panel.lovelace.editor.action-editor.default_action_help"
+          )}
+        ></ha-help-tooltip>
```

Comment:
This text and if to show the help tooltip should be passed from the parent element. The default for the button card is different from that of a light card.

Reply:
The default action for the light card will always be toggle no? If we show the default action in the list then we should show this tool tip, IMO. The desc will stay true for everything using it

Reply:
No, the way I made this, is that if there is a clear default (like for the light card toggle), the default action should be selected if you clear the action. Only when there is not a clear default it will stay at default and we should show the help icon.

At least, that's how I had it in mind...

Reply:
Committed this. Let me know if this Is what you had in mind?